core/error_test.go

```diff
@@ -35,17 +33,6 @@ func TestErrorStackTrace(t *testing.T) {
 		t.Fatal("Description did not match provided")
 	}
 
-	fileName := "error_test.go"
-	lineNum := 27 // line number where error was formed
-	funcName := "github.com/contiv/netplugin/core.getError"
-
-	expectedStr := fmt.Sprintf("%s [%s %s %d]", msg, funcName, fileName, lineNum)
-
-	if e.Error() != expectedStr {
-		t.Fatalf("Error message yielded an incorrect result with CONTIV_TRACE unset: %s", e.Error())
-	}
-
-	os.Setenv("CONTIV_TRACE", "1")
 	if e.Error() == "an error\n" {
 		t.Fatal("Error message did not yield stack trace with CONTIV_TRACE set")
```

Comment:
`CONTIV_TRACE` is not used anymore, right?

Reply:
right

Reply:
Then, why does the error say `Error message did not yield stack trace with CONTIV_TRACE set`? Should the message be corrected?